README.md

```diff
@@ -1,87 +1,129 @@
+* __Urgency__: Defaults to 3 - Low. Configurable
+* __Impact__: Defaults to 3 - Low. Configurable
+* __State__: Defaults to New. Configurable
+* __Short Description__: Contains the following information
+  * __Status__: changed, unchanged, failed, pending changes
+  * __Node__: the fqdn of the node the agent ran on
+  * __Report Time__: The timestamp of the report to help find the report in the console
+* __Description__: See the description section from event management above. Incidents will get the same description
 
-#### Parameters
+## Troubleshooting
 
-##### `meow`
+To verify that everything worked, trigger a Puppet run on one of the nodes in the `PE Master` node group then log into the node. Once logged in, run `sudo tail -n 60 /var/log/puppetlabs/puppetserver/puppetserver.log | grep servicenow`. You should see some output. If not, then the class is probably not being classified properly. Either the class is not being assigned to the Puppet server nodes at all, or there may be catalog compilation errors on those nodes with the provided parameter values. Please use GitHub issues to file any bugs you find during your troubleshooting.
 
-Enables vocalization in your cat. Valid options: 'string'.
+## Development
 
-Default: 'medium-loud'.
+### Unit tests
+To run the unit tests:
+
+```
+bundle install
+bundle exec rake spec/unit
+bundle exec rake spec/classes
 ```
 
-## Limitations
+### Acceptance tests
+The acceptance tests use puppet-litmus in a multi-node fashion. The nodes consist of a 'master' node representing the PE master (and agent), and a 'ServiceNow' node representing the ServiceNow instance. All nodes are stored in a generated `inventory.yaml` file (relative to the project root) so that they can be used with Bolt.
 
-In the Limitations section, list any incompatibilities, known issues, or other warnings.
+To setup the test infrastructure, use `bundle exec rake acceptance:setup`. This will:
 
-## Development
+* **Provision the master VM**
+* **Setup PE on the VM**
+* **Setup the mock ServiceNow instance.** This is just a Docker container on the master VM that mimics the relevant ServiceNow endpoints. Its code is contained in `spec/support/acceptance/servicenow`.
+* **Install the module on the master**
+
+Each setup step is its own task; `acceptance:setup`'s implementation consists of calling these tasks. Also, all setup tasks are idempotent. That means its safe to run them (and hence `acceptance:setup`) multiple times.
+
+**Note:** You can run the tests on a real ServiceNow instance. To do so, make sure that you've installed the event management plugin in your Servicenow instance and replaced the servicenow node in your inventory with the instance you wish to use.
```

Comment:
We should mention the required environment variables so that the rake task knows how to setup the inventory to point to a real dev instance.